src/games/strategy/triplea/ai/proAI/data/ProUnitManager.java

```diff
@@ -0,0 +1,23 @@
+package games.strategy.triplea.ai.proAI.data;
```

Comment:
Please add a javadoc to explain this class. It's a bit unclear what is going on. My best guess is we have a violation of data encapsulation and this is simply a mutable class with global variables.